src/cpr-mask.helpers.js

```diff
@@ -129,7 +131,7 @@ export function deleting(input, oldMask, maskPattern, cursor, filler) {
 
 export function checkCharsMatchPattern(pattern, chars, filler) {
 	for (let i = 0; i < pattern.length; i++) {
-		if (chars[i] === filler) return false;
+		if (chars[i] === filler && chars[i] !== pattern[i]) return false;
```

Comment:
We want to pass the check when mask characters are included in the new input value